examples/apps/ping/src/icmpv4.adb

```diff
@@ -236,10 +236,10 @@ is
 
    procedure Print (Buf : in out RFLX.RFLX_Builtin_Types.Bytes_Ptr)
    is
-      use type RFLX.ICMP.Tag;
+      use type RFLX.ICMP.Tag, RFLX.RFLX_Builtin_Types.Bit_Length;
```

Comment:
I don't think we have something in our style guide on that, but I find those comma separated with/use clauses harder to comprehend and would rather prefer them on separate lines. What do you think?

Reply:
You are right, use clauses on separate lines are easier to comprehend. I have changed it.